src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -69,7 +70,10 @@
        [quo/page-nav
         {:type       :no-title
          :background :blur
-         :right-side [{:icon-name :i/info}]
+         :right-side [{:icon-name :i/info
+                       :on-press  #(rf/dispatch [:show-bottom-sheet
+                                                 {:content (fn []
+                                                             [account-origin/view])}])}]
```

Comment:
Since we don't have any props for `account-origin`, we can pass the reference directly instead of returning a function.
```suggestion
                       :on-press  #(rf/dispatch [:show-bottom-sheet
                                                 {:content account-origin/view}])}]
```